.dockerignore

```diff
@@ -0,0 +1,17 @@
+.DS_Store
+build
+docs/_build
+__pycache__
+*.swp
+node_modules
+.project
+yarn.lock
```

Comment:
Is this not needed to insure the same set of dependencies are installed inside the container?

Reply:
The `package.json` and `package-lock.json` will ensure that the same set of dependencies is installed in the container. You don't need `node_modules` because that will be created with `npm install`. `yarn.lock` isn't used by `npm install` as far as I know.

Reply:
It's used by people using yarn... If someone wanted to switch this I wouldn't want them to have to track down this and alter it also... including it does no harm.